causing/utils.py

```diff
@@ -348,9 +348,8 @@ def vecmat(mz):
 
 def render_dot(dot_str, filename):
     """render Graphviz graph from dot_str to svg"""
-    filename = str(filename).split(".svg")[0]
-    data = graphviz.Source(dot_str)
-    data.render(filename=filename, format="svg", cleanup=True)
+    data = pydot.graph_from_dot_data(dot_str)
```

Comment:
So we just need pydot to create the graphs?

Reply:
Yes, we can create graphs using pydot only.